integration/kubernetes/init.sh

```diff
@@ -18,7 +18,7 @@ set -e
 SCRIPT_PATH=$(dirname "$(readlink -f "$0")")
 source "${SCRIPT_PATH}/lib.sh"
 
-sudo -E kubeadm init --pod-network-cidr 10.244.0.0/16
+sudo -E kubeadm init --pod-network-cidr 10.244.0.0/16 --ignore-preflight-errors=CRI
```

Comment:
why it is need to ignore CRI errors ?

Reply:
Because kubelet searches for `dockershim.sock`. As we do not use that socket and instead we use CRI-O, we need to ignore it.

Reply:
oh thats true, I had the sam problem, I added `--cri-socket`
kubeadm init --pod-network-cidr=10.244.0.0/16 --cri-socket /var/run/crio.sock

Reply:
Same for reset:
kubectl drain $(hostname) --delete-local-data --force --ignore-daemonsets
kubectl delete node --all
kubeadm reset  --cri-socket /var/run/crio.sock